File: Naloga 32/naloga_32.py
def naloga_32():
    stevilo = int(input("stavi stevilo: "))
    for delitelj in range(1, stevilo):
        if not (stevilo % delitelj):
            print(delitelj)
    print(stevilo)
    return
if __name__ == '__main__':
    naloga_32()

File: Naloga 32/naloga_32.cpp
#include <iostream>

void naloga_32 () {
    int stevilo;
    std::cin >> stevilo;
    for (int delitelj=1; delitelj <=stevilo; ++delitelj) {
        if (!(stevilo%delitelj)) {
            std::cout << delitelj << std::endl;
        };
    };
};

int main () {
    naloga_32();
    return 0;
};
